lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -134,7 +137,12 @@ serveWallet (cfg, tr) databaseDir hostPref listen lj beforeMainLoop = do
             let (_, bp) = staticBlockchainParameters nl
             rndApi <- apiLayer tr (toWLBlock <$> nl)
             seqApi <- apiLayer tr (toWLBlock <$> nl)
-            startServer tr nPort bp rndApi seqApi
+
+            -- StakePool
+            (_, spDB) <- PoolSqlite.newDBLayer cfg tr (poolDBPath databaseDir)
```

Comment:
We should use `withDBLayer` here, actually I don't even know why we have a `newDBLayer` still exported and available...

Reply:
Done, I don't know why, I expected it to be more complicated.

Reply:
I think it's actually even simpler... :sweat_smile: